app/controllers/AnnotationIOController.scala

```diff
@@ -491,7 +518,10 @@ Expects:
       project <- projectDAO.findOne(projectIdValidated) ?~> Messages("project.notFound", projectId) ~> NOT_FOUND
       _ <- Fox.assertTrue(userService.isTeamManagerOrAdminOf(user, project._team)) ?~> "notAllowed" ~> FORBIDDEN
       annotations <- annotationDAO.findAllFinishedForProject(projectIdValidated)
-      zip <- annotationService.zipAnnotations(annotations, project.name, skipVolumeData)
+      zip <- annotationService.zipAnnotations(annotations,
```

Comment:
The Task system can be used to ask users to annotate certain areas of datasets in certain ways. Tasks are organized in projects. Users can then request a task and webknossos will automatically assign them.

These routes here allow the organization’s admins/team managers to download the result of such task annotations (or collections of such task annotations, e.g. grouped by project).

Eventually we want to enable ND datasets for this as well. However, since the UI does not currently supply format selection methods for these download routes, and since not every client can handle the new format yet, we decided that for now these features should stay with the WKW format, even if that means that they cannot be used productively with ND data.